core/tests/protractor_desktop/playVoiceovers.js

```diff
@@ -0,0 +1,104 @@
+    explorationEditorTranslationTab =
+      explorationEditorPage.getTranslationTab();
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    libraryPage = new LibraryPage.LibraryPage();
+
+    await users.createUser('testVoiceovers@voiceovers.com',
+      'testVoiceovers');
+    await users.login('testVoiceovers@voiceovers.com');
+    await workflow.createExploration();
+    await explorationEditorMainTab.exitTutorial();
+    await explorationEditorMainTab.setStateName('First');
+    await explorationEditorMainTab.setContent(await forms.toRichText(
+      'This is the first card.'));
+    await explorationEditorMainTab.setInteraction('EndExploration');
+    await explorationEditorPage.navigateToTranslationTab();
+    await explorationEditorTranslationTab.uploadAudioFileForLanguage(
+      'Hindi', '../data/cafe.mp3');
+    await explorationEditorPage.saveChanges();
+    await explorationEditorPage.navigateToSettingsTab();
+    await explorationEditorSettingsTab.setTitle('voiceoverPlayerTest');
+    await explorationEditorSettingsTab.setCategory('Languages');
+    await explorationEditorSettingsTab.setLanguage('English');
+    await explorationEditorSettingsTab.setObjective(
+      'Testing if voiceovers work');
+    await explorationEditorPage.saveChanges('Done.');
+    await workflow.publishExploration();
+    await users.logout();
+  });
+
+  it('should play and pause voiceovers', async function() {
+    await users.login('testVoiceovers@voiceovers.com');
+    await libraryPage.get();
+    await libraryPage.playExploration('voiceoverPlayerTest');
+    await explorationPlayerPage.expandAudioBar();
+    await explorationPlayerPage.pressPlayButton();
+    await explorationPlayerPage.expectAudioToBePlaying();
+    await explorationPlayerPage.pressPauseButton();
+    await explorationPlayerPage.expectAudioToBePaused();
+  });
+
+  it('should play voiceovers for multiple languages', async function() {
+    await creatorDashboardPage.get();
+    await creatorDashboardPage.editExploration('voiceoverPlayerTest');
+    await explorationEditorPage.navigateToTranslationTab();
+    await explorationEditorTranslationTab.uploadAudioFileForLanguage(
+      'Arabic', '../data/ambient-noise.mp3');
+    await explorationEditorPage.saveChanges('Added another voiceover');
+    await libraryPage.get();
+    await libraryPage.playExploration('voiceoverPlayerTest');
+    await explorationPlayerPage.expandAudioBar();
+    await explorationPlayerPage.changeVoiceoverLanguage('Hindi');
+    await explorationPlayerPage.pressPlayButton();
+    await explorationPlayerPage.expectAudioToBePlaying();
+    await explorationPlayerPage.pressPauseButton();
+    await explorationPlayerPage.changeVoiceoverLanguage('Arabic');
+    await explorationPlayerPage.pressPlayButton();
+    await explorationPlayerPage.expectAudioToBePlaying();
+  });
```

Comment:
Are we missing `afterEach` section which will have `no console error check`?

Reply:
Adding `general.checkForConsoleErrors([])` to the `afterEach` section throws an exception: `The play() request was interrupted by a call to pause()`. I've emailed @U8NWXD about this issue in the past. I deleted the console error check to resolve the issue.

Is it necessary to add `general.checkForConsoleErrors([])` in this case?

Reply:
You can give this particular exception in the list of error messages which we allow (I think this is implemented within that function). This check will help us make sure that there are no new error messages.